common/v2/components/TransactionFlow/TxReceipt.tsx

```diff
@@ -42,13 +43,26 @@ import sentIcon from 'common/assets/images/icn-sent.svg';
 import defizaplogo from 'assets/images/defizap/defizaplogo.svg';
 import './TxReceipt.scss';
 
+const PendingTransaction: FC = () => {
+  return (
+    <div className="pending-loader">
+      <div className="loading" />
+      <span>
+        <b>Pending</b>
+      </span>
+    </div>
+  );
+};
```

Comment:
I will just move it, the "loading" class in global, I would also suggest using this one for all loaders, since it can be attached to any button, div containers. Looks really good.